docs/data/stashcache/install-origin.md

```diff
@@ -5,15 +5,15 @@ This document describes how to install a StashCache origin service.  This servic
 to export its data to the StashCache data federation.
 
 !!! note
-    The origin must be registered with the OSG prior to joining the data federation.  You may start the
-    registration process prior to finishing the installation by contacting <support@opensciencegrid.org>
-    with the following details:
+    The _origin_ must be registered with the OSG prior to joining the data federation. You may start the
+    registration process prior to finishing the installation by [using this link](#registering-the-origin) 
+    along with the basic information like:
 
     * Resource name and hostname.
     * VO associated with this origin server (will be used to determine the origin's namespace prefix).
     * Administrative and security contact.
 
-    Follow the [registration documentation](/common/registration.md) for more information.
+    Detailed walkthrough of the process is described [here](#registering-the-origin). 
```

Comment:
You've got two links to the same place a few lines apart; I don't think you need both.